test/apis/authentication-api.ts

```diff
@@ -165,6 +192,33 @@ describe('Authentication API', () => {
     assert(currentPatient.lastName == 'Duchâteau')
   }).timeout(60000)
 
+  it('Patient should be able to signing up through email with friendlyCaptchaData', async () => {
+    // When
+    const recaptcha =
```

Comment:
From where does this recaptcha comes from ?
For such sensitive elements, put them in an environment variable, to avoid having them in clear in the code

Reply:
Regenerating at each login so no problem

Reply:
This is an example of the Friendly Captcha answer. It is regenerated every time we reload the captcha.